### PR TITLE
Minor samples repo cleanup

### PR DESCRIPTION
**Pull request by chcosta**

### CODE REVIEW

File: HelloWorld/HelloWorld.csproj [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>netcoreapp2.0</TargetFrameworks>
```
Comment (jonfortescue) [inline HelloWorld/HelloWorld.csproj]: wow i hate that this was the problem

File: build.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
```
Comment (jonfortescue) [inline build.sh]: So currently we don't reference this anywhere &ndash; is it a common problem to not have bash be in `/bin/bash`?

Comment (jonfortescue) [reply build.sh]: this, being this script -- I don't see it being used anywhere is what I'm saying

Comment (chcosta) [reply build.sh]: It's just a common entry point for a dev to use to manually build a repo.  The same way this build.cmd is present in dotnet/arcade.  Nothing automated is referencing it, but it's useful for anybody manually building the repo or looking to this repo as an example of how to setup their repo.

Comment (jonfortescue) [reply build.sh]: gotcha. makes sense to me.

**Pull request merged**